src/main/java/gregtech/common/blocks/BlockBatteryPart.java

```diff
@@ -0,0 +1,88 @@
+package gregtech.common.blocks;
+
+import gregtech.api.GTValues;
+import gregtech.api.block.VariantBlock;
+import gregtech.api.metatileentity.multiblock.IBatteryData;
+import net.minecraft.block.SoundType;
+import net.minecraft.block.material.Material;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.client.resources.I18n;
+import net.minecraft.client.util.ITooltipFlag;
+import net.minecraft.entity.EntityLiving;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.IStringSerializable;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.IBlockAccess;
+import net.minecraft.world.World;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.List;
+
+public class BlockBatteryPart extends VariantBlock<BlockBatteryPart.BatteryPartType> {
+
+    public BlockBatteryPart() {
+        super(Material.IRON);
+        setTranslationKey("battery_block");
+        setHardness(5.0f);
+        setResistance(10.0f);
+        setSoundType(SoundType.METAL);
+        setDefaultState(getState(BatteryPartType.EMPTY_TIER_I));
+    }
+
+    @Override
+    public boolean canCreatureSpawn(@NotNull IBlockState state, @NotNull IBlockAccess world, @NotNull BlockPos pos, @NotNull EntityLiving.SpawnPlacementType placementType) {
+        return false;
+    }
+
+    @Override
+    public void addInformation(@NotNull ItemStack stack, @Nullable World world, List<String> tooltip, @NotNull ITooltipFlag advanced) {
+        super.addInformation(stack, world, tooltip, advanced);
+
+        BatteryPartType batteryType = getState(stack);
+        if (batteryType.getCapacity() != 0) {
+            tooltip.add(I18n.format("gregtech.universal.tooltip.energy_storage_capacity", batteryType.getCapacity()));
+        } else {
+            tooltip.add(I18n.format("tile.battery_block.tooltip_empty"));
+        }
+    }
+
+    public enum BatteryPartType implements IStringSerializable, IBatteryData {
+        EMPTY_TIER_I(-1, 0),
```

Comment:
Would be better to have a second ctor for this, so the "empty" behavior is unified.